content/blog/2016/2016-08-30-ask-experts-demos.adoc

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Demos at Jenkins World 2016"
+tags:
+- jenkinsworld
+- event
+author: rtyler
+---
+
+image:/images/conferences/Jenkins-World_125x125.png[role=right]
+
+At this year's link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World],
+our events officer link:https://github.com/alyssat[Alyssa] has been working to
+organize various activities in the "Open Source Hub" on the expo floor. One of
+the activities that I am personally most excited for are the Jenkins demos in
+our "open source theater." Both days of the conference (Sept. 14th and 15th),
+during the break for lunch, there will be 15 minute demos by many of the
+link:/blog/2016/08/26/ask-the-experts-jenkins-world/[experts] helping to staff
+the Open Source Hub.
+
+== Demo Schedule
+
+=== Wednesday, September 14th
+[options=header]
+|===
+| Time | Session | Details | Presenter
+| 12:15 - 12:30
+| *Blue Ocean in Action*
+| _Showcase of Blue Ocean and how it will make Jenkins a pleasure to use._
+| link:https://github.com/kzantow[Keith Zantow]
+
+| 12:30 - 12:45
+| *Notifications with Jenkins Pipeline*
+| _Sending information to Slack, HipChat email and more from your Pipeline_
+| link:https://github.com/bitwiseman[Liam Newman]
+
+| 12:45 - 13:00
+| *Docker and Pipeline*
+| _Learn how to use Docker inside of Pipeline for clean, repeatable testing environments_
+| link:https://github.com/rtyler[R Tyler Croy]
+
+| 13:00 - 13:15
+| *Git plugin - large repos, submodule authentication and more*
+| _Techniques for managing large git repositories, Submodule authentication, Pipelines and Git_
+| link:https://github.com/markewaite[Mark Waite]
+
+| 13:15 - 13:30
+| *Freestyle to Pipeline*
+| _Overview on how easy it is to migrate from a confusing series of Freestyle Jobs to Jenkins Pipeline_
+| link:https://github.com/rtyler[R Tyler Croy]
+
+| 13:30 - 13:45
+| *package.json and Jenkins*
```

Comment:
I think grammar generally dictates that in this context it must be capitalized, because you always capitalize the first word in a sentence. Subsequent uses can be downcase. But I don't care very much, if you prefer downcase no judge would convict you.